microsoft-edge/dev-videos/index.md

```diff
@@ -30,265 +30,837 @@ To add a new video:
 
 
 <!-- ====================================================================== -->
-## DevTools - What's New in DevTools 105
+## Microsoft Edge channel at YouTube
 
-_Published on 6 September 2022._
+You can find all videos about development using Microsoft Edge at the [Microsoft Edge (@MSFTEdge) channel at YouTube](https://www.youtube.com/@MSFTEdge/videos).
+
+The Microsoft Edge channel includes the following playlists:
+* [Developer](https://www.youtube.com/playlist?list=PL4z1-7pjJU6zvxxsa2GzzuaXf1neBzrmI)
+* [What's New in DevTools](https://www.youtube.com/playlist?list=PL4z1-7pjJU6wCla3QZuWuWjsCB2hCnTvr)
+* [Extensions](https://www.youtube.com/playlist?list=PL4z1-7pjJU6ykO2iptfxT7T68W5WEK4ff)
+
+
+<!-- ====================================================================== -->
+## DevTools - What's New in DevTools
```

Comment:
I see you've changed the order of the article by grouping videos in buckets: What's New in DevTools, other DevTools videos, Web Platform, etc.
Something to consider: the What's New videos are what we have the most of, which means that other buckets are always going to be very far down the article.
The advantage of sorting by date, and ignoring the categories, is you always have the latest video up top.
If we were to publish a video about a new Web Platform feature soon, then that video would be way down, not visible without scrolling a lot (and not visible in the **In this article** TOC without clicking **Show 18 more** first).

Reply:
ordered by date